src/hotspot/cpu/aarch64/aarch64.ad

```diff
@@ -2377,7 +2377,7 @@ int Matcher::min_vector_size(const BasicType bt) {
   return MIN2(size, max_size);
 }
 
-int Matcher::superword_max_vector_size(const BasicType bt) {
+int Matcher::max_vector_size_autovectorization(const BasicType bt) {
```

Comment:
Should we use two separate words like `auto_vectorization`?

Reply:
Done :)